### PR TITLE
🐛 PRTL-1337 use fetched transcripts if none provided

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: src/packages/@ncigdc/containers/Lolliplot.js [hunk 1]
```diff
@@ -773,7 +777,7 @@ export const LolliplotQuery = {
             symbol
             canonical_transcript_id
             transcripts {
-              hits(first: 1 filters: $isCanonical) {
+              hits(first: 99) {
```
Comment (shanewilson) [inline src/packages/@ncigdc/containers/Lolliplot.js]: what does this fix?

Comment (alex-wilmer) [inline src/packages/@ncigdc/containers/Lolliplot.js]: transcript dropdown above the viewer

Comment (shanewilson) [inline src/packages/@ncigdc/containers/Lolliplot.js]: what would be the real solution to this? another api call?

Comment (alex-wilmer) [inline src/packages/@ncigdc/containers/Lolliplot.js]: a paginated dropdown?

**Pull request merged**